### DISCUSSION TITLE
Error: The process '/opt/hostedtoolcache/node/14.20.0/x64/bin/npx' failed with exit code 1

### DISCUSSION THREAD

**Discussion by AlanOchMtz** [ArtiomTr/jest-coverage-report-action#304] [Q&A]
After Updating the Jest version Im getting this error:
`Error: The process '/opt/hostedtoolcache/node/14.20.0/x64/bin/npx' failed with exit code 1`
Even try to change the node version but still failling any ideas?

**Comment by ArtiomTr** [comment]
Answered in another discussion, duplicating answer here:

https://github.com/ArtiomTr/jest-coverage-report-action/discussions/249#discussioncomment-3421845

> Hello @AlanOchMtz 👋,
> 
> Could you please attach a screenshot of the action console? There are many reasons why jest exits with non-zero code. As you can see from the question above, tests succeed locally, but failed in the workflow, because there was an unresolved promise. That is a frequent issue with asynchronous tests.

**Reply by AlanOchMtz** [reply]
@ArtiomTr 
![image](https://user-images.githubusercontent.com/100713915/185414113-12455509-ce84-4869-8fa2-b3c7b86437b5.png)

What do you mean with action console?  I send you the test_coverage file Im using to run the tests

![image](https://user-images.githubusercontent.com/100713915/185448085-8135f400-73f7-4bdd-b603-449080a6db70.png)

![image](https://user-images.githubusercontent.com/100713915/185448230-7fd08f81-4dfb-4d96-8f6b-9f455bbc5a91.png)

![image](https://user-images.githubusercontent.com/100713915/185449174-abffce9b-40c2-4022-935f-29ebb0b6e8c4.png)

**Reply by ArtiomTr** [reply]
@AlanOchMtz, as I can see, your code is producing UnhandledPromiseRejection warnings - they could potentially cause exiting with non-zero exit code. Consider fixing all the warnings.
